### DISCUSSION TITLE
It is asking ELEVATED access all the time

### DISCUSSION THREAD

**Discussion by iashok22** [tweepy/tweepy#1815] [Questions] [answered]
Recently, I have tried to integrate with my bot with regular privilege but tweepy was not able to authenticate or do any action and it was asking me to get ELEVATED access in order to work but in the meantime I was able to hit v2 API's from the Postman to tweet successfully. Does it mean that Tweepy is not updated to support V2 endpoints?

**Comment by Harmon758** [comment, marked as answer]
Tweepy supports Twitter API v2. See #1472.
You're likely using [`API`](https://tweepy.readthedocs.io/en/v4.5.0/api.html), the Twitter API v1.1 interface in Tweepy, rather than [`Client`](https://tweepy.readthedocs.io/en/v4.5.0/client.html), the Twitter API v2 interface.
If that's not the case, you'll need to provide the relevant code and/or full traceback for further help.
